File: Difficulty: Basic/Min and Max in Array/min-and-max-in-array.java
class Solution {
    public ArrayList<Integer> getMinMax(int[] arr) {
        // code Here
        ArrayList<Integer> res= new ArrayList<>();
        int min=Integer.MAX_VALUE;
        int max=Integer.MIN_VALUE;
        for(int i=0;i<arr.length;i++){
            if(min>arr[i]){
                min=arr[i];
                
            }
            if(max<arr[i]){
                max=arr[i];
            }
        }
        res.add(min);
        res.add(max);
        return res;
    }
}

File: Difficulty: Basic/Min and Max in Array/min-and-max-in-array.py
class Solution:
    def getMinMax(self, arr):
        # code here
        min=arr[0]
        max=arr[0]
        i=0
        n=len(arr)
        for i in range(1,n):
            if arr[i]<min:
                min=arr[i]
            if arr[i]>max:
                max=arr[i]
            i=i+1
        
        return [min,max]